src/components/ServiceSearch.jsx

```diff
@@ -15,11 +15,11 @@ import {
   InputLabel,
   MenuItem,
   Hidden,
-  useMediaQuery
+  useMediaQuery,
```

Comment:
Formatter conflicts?

Reply:
no shouldn't be. probably wasn't formatted before